tests/browser_test_case.py

```diff
@@ -0,0 +1,79 @@
+import os
+import time
+
+from selenium.webdriver.chrome.options import Options
+from selenium import webdriver
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+
+from dataload.import_to_elasticsearch import import_to_elasticsearch
+
+# Test data directory
+prefix = os.path.join(os.path.dirname(__file__), "data")
+
+
+class BrowserTestCase(StaticLiveServerTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        import_to_elasticsearch(
+            [
+                os.path.join(prefix, "a002400000KeYdsAAF-currency.json"),
+                os.path.join(prefix, "a002400000OiDBQAA3.json"),
+                os.path.join(prefix, "awefw001p00000zgyHZAAY.json"),
+            ],
+            clean=True,
+            funders=os.path.join(prefix, "funders.jsonl"),
+            recipients=os.path.join(prefix, "recipients.jsonl"),
+        )
+        # elastic search needs some time to commit its data
+        time.sleep(2)
+
+    def setUp(self, *args, **kwargs):
+        BROWSER = os.environ.get("BROWSER", "ChromeHeadless")
+
+        if BROWSER == "ChromeHeadless":
+            chrome_options = Options()
+            chrome_options.add_argument("--headless")
+            # no-sandbox prevents an error when running as the root user
+            chrome_options.add_argument("--no-sandbox")
+            # uncomment this if "DevToolsActivePort" error / ubuntu snap workaround
+            chrome_options.add_argument("--remote-debugging-port=9222")
+            # chrome_options.add_argument('ignore-unexpected-deprecations')
+            self.browser = webdriver.Chrome(options=chrome_options)
+        elif BROWSER == "Firefox":
+            # Make downloads work
+            profile = webdriver.FirefoxProfile()
+            profile.set_preference("browser.download.folderList", 2)
+            profile.set_preference("browser.download.manager.showWhenStarting", False)
+            profile.set_preference("browser.download.dir", os.getcwd())
+            profile.set_preference(
+                "browser.helperApps.neverAsk.saveToDisk", "application/json"
+            )
+            self.browser = getattr(webdriver, BROWSER)(firefox_profile=profile)
```

Comment:
Running the tests with Firefox no longer works for me, with error:
```
Traceback (most recent call last):
  File "/home/rikki/Projects/grantnav/tests/browser_test_case.py", line 54, in setUp
    self.browser = getattr(webdriver, BROWSER)(firefox_profile=profile)
TypeError: __init__() got an unexpected keyword argument 'firefox_profile'
```
... but let's fix that later, just a "nice to have"

Reply:
Probably just due to upgrades?